
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_zhiding(object):
    def setupUi(self, zhiding):
        zhiding.setObjectName("zhiding")
        zhiding.resize(924, 650)
        self.tableView = QtWidgets.QTableView(zhiding)
        self.tableView.setGeometry(QtCore.QRect(30, 40, 256, 541))
        self.tableView.setObjectName("tableView")
        self.tableView_2 = QtWidgets.QTableView(zhiding)
        self.tableView_2.setGeometry(QtCore.QRect(350, 40, 256, 541))
        self.tableView_2.setObjectName("tableView_2")
        self.label = QtWidgets.QLabel(zhiding)
        self.label.setGeometry(QtCore.QRect(30, 20, 101, 16))
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(zhiding)
        self.label_2.setGeometry(QtCore.QRect(350, 20, 121, 16))
        self.label_2.setObjectName("label_2")
        self.pushButton = QtWidgets.QPushButton(zhiding)
        self.pushButton.setGeometry(QtCore.QRect(730, 410, 93, 28))
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(zhiding)
        self.pushButton_2.setGeometry(QtCore.QRect(730, 490, 93, 28))
        self.pushButton_2.setObjectName("pushButton_2")
        self.checkBox_6 = QtWidgets.QCheckBox(zhiding)
        self.checkBox_6.setGeometry(QtCore.QRect(670, 110, 161, 19))
        self.checkBox_6.setObjectName("checkBox_6")
        self.checkBox_7 = QtWidgets.QCheckBox(zhiding)
        self.checkBox_7.setGeometry(QtCore.QRect(670, 150, 161, 19))
        self.checkBox_7.setObjectName("checkBox_7")
        self.checkBox_8 = QtWidgets.QCheckBox(zhiding)
        self.checkBox_8.setGeometry(QtCore.QRect(670, 190, 161, 19))
        self.checkBox_8.setObjectName("checkBox_8")
        self.checkBox_9 = QtWidgets.QCheckBox(zhiding)
        self.checkBox_9.setGeometry(QtCore.QRect(670, 270, 161, 19))
        self.checkBox_9.setObjectName("checkBox_9")
        self.checkBox_10 = QtWidgets.QCheckBox(zhiding)
        self.checkBox_10.setGeometry(QtCore.QRect(670, 310, 161, 19))
        self.checkBox_10.setObjectName("checkBox_10")
        self.checkBox_11 = QtWidgets.QCheckBox(zhiding)
        self.checkBox_11.setGeometry(QtCore.QRect(670, 230, 161, 19))
        self.checkBox_11.setObjectName("checkBox_11")

        self.retranslateUi(zhiding)
        QtCore.QMetaObject.connectSlotsByName(zhiding)

    def retranslateUi(self, zhiding):
        _translate = QtCore.QCoreApplication.translate
        zhiding.setWindowTitle(_translate("zhiding", "Form"))
        self.label.setText(_translate("zhiding", "今天上课学生"))
        self.label_2.setText(_translate("zhiding", "特定时间上课学生"))
        self.pushButton.setText(_translate("zhiding", "增加"))
        self.pushButton_2.setText(_translate("zhiding", "删除"))
        self.checkBox_6.setText(_translate("zhiding", "8：00-10：00上"))
        self.checkBox_7.setText(_translate("zhiding", "10：00-12：00上"))
        self.checkBox_8.setText(_translate("zhiding", "12：00-2：00上"))
        self.checkBox_9.setText(_translate("zhiding", "4：00-6：00上"))
        self.checkBox_10.setText(_translate("zhiding", "7：00-9：00上"))
        self.checkBox_11.setText(_translate("zhiding", "2：00-4：00上"))
